sandbox.go

```diff
@@ -133,7 +133,8 @@ type containerConfig struct {
 }
 
 const (
-	resolverIPSandbox = "127.0.0.11"
+	resolverIPSandbox  = "127.0.0.11"
```

Comment:
The additional space is an unrelated change.

Reply:
This space was produced by gofmt, are you sure it should be as you said?

Reply:
It doesn't matter who or what produced it. It's a change that doesn't belong to the title of this PR. As such, it will distract any reviewers of this PR.

Reply:
If i'll remove it - 
```
🐳 vet
🐳 ineffassign

🐳 fmt
sandbox.go
👹 please format Go code with 'gofmt -s -w'
Makefile:155: recipe for target 'fmt' failed
make: *** [fmt] Error 1
Makefile:114: recipe for target 'check' failed
make: *** [check] Error 2
Exited with code 2
```

Reply:
Another advice, though I don't know whether this applies to docker specifically, but you usually update the pull request, rather than adding more commits that fix the problem in the original pull request.
The reason is to keep the repository history clean and concise.
What I mean, it's better to have a commit that implements the desired functionality (or a logical part of it) without any unrelated changes such as code reformatting, and then another commit that does the reformatting. Rather than having a commit that implements the functionality including unrelated changes, than a commit that reverts the unrelated changes, and finally a commit that adds them back like here.

But to end on a more positive note - I really hope this will get merged soon (whatever version will be accepted by the devs). It's an incredibly useful functionality.

Reply:
Totally agree with you about the history and will keep in mind for future
;)